Packs/ApiModules/Scripts/MLApiModule/MLApiModule.py

```diff
@@ -0,0 +1,237 @@
+from CommonServerPython import *
+from CommonServerUserPython import *
+import spacy
+import string
+from html.parser import HTMLParser
+from html import unescape
+from re import compile as _Re
+import pandas as pd
+
+
+def handle_long_text(t, input_length):
+    if input_length == 1:
+        return_error("Input text length ({}) exceeds the legal maximum length for preprocessing".format(len(t)))
```

Comment:
why we return error in that case?
I think return error do sys.exit, do we want that?